### DISCUSSION TITLE
Autel Evo II 640T "not enough bits"

### DISCUSSION THREAD

**Discussion by jnazario1** [bluenviron/mediamtx#2829] [Questions]
### Question

Anyone encounter any issues trying to stream an autel drone via RTMP? I have been working on trying to stream an autel evo II 640t but I am getting "not enough bits" when I run "systemctl status mediamtx". The udpmaxpayload size is already set at 1472 and I have tried playing around with the queue size but nothing changes. Any help will be greatly appreciated. This is running on an 8gb raspberry pi 4.

**Comment by aler9** [comment]
Please open a bug report and provide details that allow to identify the issue, including a network dump and server logs, by following instructions.
